tune_sklearn/tune_basesearch.py

```diff
@@ -291,14 +291,16 @@ def _fit(self, X, y=None, groups=None, **fit_params):
 
         if self.n_jobs is not None:
             if self.n_jobs < 0:
+                available_cpus = multiprocessing.cpu_count()
                 resources_per_trial = {
-                    "cpu": max(multiprocessing.cpu_count() + 1 + self.n_jobs,
-                               1),
+                    "cpu": 1,
                     "gpu": 1 if self.use_gpu else 0
                 }
+                warnings.warn("`self.n_jobs` is automatically set "
+                              "-1 for any negative values.")
             else:
                 resources_per_trial = {
-                    "cpu": self.n_jobs,
+                    "cpu": available_cpus / self.n_jobs,
```

Comment:
i don't think available_cpus is defined in this scope?

Reply:
Good catch